render-tests/generate_pmtiles.sh

```diff
@@ -1,9 +1,38 @@
+#!/bin/bash
+set -euo pipefail
+
+target_subdir="${1:-}"
+
+if [[ -n "$target_subdir" ]]; then
+  echo "Processing only subfolder: $target_subdir"
+  mapfile -t input_files < <(find "$target_subdir" -type f -name "input.osm.pbf")
```

Comment:
`mapfile` doesn't exist on macOS, is there an alternative?

Reply:
Also, if you think this file is too much of a mess or is hard to maintain, I am happy to write the functionality in typescript instead...